lib/game_box_web/live/arena_live.ex

```diff
@@ -47,113 +48,182 @@ defmodule GameBoxWeb.ArenaLive do
 
     ~H"""
     <%= if board == "" do %>
-      <.hero
-        subheader="Arena"
-        header={@arena.arena_id}
-        subtext={populate_subtext(@game_selected, @is_host)}
-      />
       <%= if is_nil(@game_selected) do %>
-        <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-x-12 gap-y-12 mb-12">
-          <%= for game <- @games do %>
-            <.card>
-              <.card_media :if={game.artwork} src={game.artwork} />
-              <.card_media
-                :if={!game.artwork}
-                src="/images/donut.png"
-                class="flex justify-center w-48 p-6"
-              />
-              <.card_content
-                author={"@#{game.user.gh_login}"}
-                author_link={"https://github.com/@#{game.user.gh_login}"}
-                heading={game.title}
-              />
-              <.card_footer>
-                <%= if @is_host do %>
-                  <.button
-                    phx-click="select_game"
-                    phx-value-game_id={game.id}
-                    label="Start"
-                    class="w-full"
+        <div class="w-full mb-6">
+          <.card>
+            <.card_content>
+              <div class="flex flex-col md:flex-row md:mb-6 justify-start items-start">
+                <div class="w-full md:w-1/3  md:border-r md:border-zinc-700 mr-6">
+                  <.arena_code arena_id={@arena.arena_id} />
+                </div>
+                <div class="w-2/3">
+                  <div class="mb-6 pt-12 md:pt-0">
+                    <.p class="font-display tracking-wider uppercase text-xs !mt-0 !pt-0 !pb-2">
+                      Players Online <span class="text-md">(<%= @total_players %>)</span>
+                    </.p>
+                    <.ol class="grid grid-cols-1 md:grid-cols-2 gap-x-12">
+                      <li :for={player <- @all_players}>
+                        <%= player.name %>
+                        <%= check_if_host(Arena.get_host(@arena.arena_id), player.id) %>
```

Comment:
could this be `@is_host` instead of using the Arena.get_host function?

Reply:
I don't think so, unless I might be missing something. It looks to me like `is_host` is a boolean and it only has to do with if the current person is the active host. This is checking the current host against the player list so _everyone_ can see who the host is, because I think that's relevant information to everyone playing the game.

Reply:
Gotcha. I think we have this value already in the arena state which is a variable on the socket? Maybe we could use that instead of calling this function a few times and pattern match it out of assigns in our `check_if_host` function

Reply:
Ahhh cool cool, got that fixed up. 👍